hash.c

```diff
@@ -6881,93 +6881,161 @@ static const rb_data_type_t env_data_type = {
  *
  *    reviews.length #=> 1
  *
- *  === Default Values
+ *  === Key Not Found?
```

Comment:
I find a question in a title a bit odd. How about:

```suggestion
 *  === Nonexistent keys
```

Reply:
> I find a question in a title a bit odd. How about:

I like this title;  it captures the spirit of what I intend for the section.  So, stet?